website/content/docs/internals/high-availability.mdx

```diff
@@ -47,7 +47,41 @@ the request, the request is forwarded to the active server. Read-only requests a
 Like traditional HA standbys, a Performance Standby Node becomes the active instance when the active node is sealed, fails, or loses
 network connectivity.
 
-# Tutorial 
+# Becoming an active node
+
+The active node in an HA configuration is the node that holds an HA lock. When a node relinquishes active duty, either
+because it's unavailable or because an operator has invoked [`vault operator step-down`](/vault/docs/commands/operator/step-down),
+all the nodes in the cluster will attempt to grab the HA lock. Only one node will succeed and this node will become the new active
+node.
+
+How the HA lock works is dependent on which HA storage backend the cluster is running. In Integrated Storage, the HA
+lock is always given to the node which wins the [Raft leadership election](/vault/docs/internals/integrated-storage#leadership-elections).
+
+Once a node has grabbed the HA lock it will perform a number of setup steps. The node will:
+* Seal
+* Reload the seal configuration
+* Migrate the seal, if needed
+* Reload keys
+* Create a new HA intra-cluster TLS certificate
+* Write an entry to storage advertising that this node is the active node
+
+Finally, the new active node will unseal and perform its post-unseal setup tasks. In the post-unseal tasks, this new
+active node will support receiving connections from the other nodes in the cluster. Once post-unseal setup completes
+the node will be able to respond to new client requests.
```

Comment:
```suggestion
After obtaining an HA lock, the node goes through a series of inauguration steps
to formally become the active node for its cluster:

1. Seals the local Vault instance.
1. Reloads the seal configuration.
1. Migrates the seal, if needed.
1. Reloads encryption keys.
1. Creates a new HA intra-cluster TLS certificate.
1. Writes an entry to storage advertising its status as the active node.
1. Unseals the local Vault instance.
1. Start accepting connections from other nodes in the cluster.

After completing the inauguration steps, the new active node beings responding to new 
client requests.
```

We should be specific about what seals and what kind of keys we're talking about (I made some guesses for the suggested edit)

Style correction: write in active voice.